app/views/posts/index.html.erb

```diff
@@ -1,41 +1,32 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Posts</h1>
+<div class="container d-flex">
+  <div class="row align-self-center"><h2>Posts</h2></div>
+</div>
 
-<table>
-  <thead>
-    <tr>
-      <th>Title</th>
-      <th>Body</th>
-      <th>Author</th>
-      <th>Replies count</th>
-      <th>Votes count</th>
-      <th>Favorites count</th>
-      <th>Follows count</th>
-      <th>Tags count</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
+<div class="container">
+  <% @posts.each do |post| %>
+      <div class="d-flex border-top">
+        <div class="pt-4 h5 text-info"><%= post.title %></div>
+      </div>
+      <div class="pb-2 d-flex">
+        <div><%= post.body %></div>
+      </div>
 
-  <tbody>
-    <% @posts.each do |post| %>
-      <tr>
-        <td><%= post.title %></td>
-        <td><%= post.body %></td>
-        <td><%= post.author_id %></td>
-        <td><%= post.replies_count %></td>
-        <td><%= post.votes_count %></td>
-        <td><%= post.favorites_count %></td>
-        <td><%= post.follows_count %></td>
-        <td><%= post.tags_count %></td>
-        <td><%= link_to 'Show', post %></td>
-        <td><%= link_to 'Edit', edit_post_path(post) %></td>
-        <td><%= link_to 'Destroy', post, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+        <div class="d-flex flex-row-reverse">
+          <div><%= image_tag post.author.avatar_img, style: "height:30px"%></div>
+          <div><small class="text-muted">Posted on <%= post.created_at.strftime("%m-%d-%Y") %> at <%= post.created_at.strftime("%I:%M:%S") %> by <%= post.author.username %></small></div>
+        </div>
 
-<br>
 
-<%= link_to 'New Post', new_post_path %>
+      <% post.replies.each do |reply| %>
+        <div class="pt-1 pl-4 d-flex">
+          <div class=""><%= reply.body %></div>
+        </div>
+        <div class="pl-5 pb-2 d-flex flex-row">
+          <div class="align-self-center"><%= image_tag reply.author.avatar_img, style: "height:20px"%></div>
+          <div class="align-self-center"><small class="text-muted">Posted on <%= post.created_at.strftime("%m-%d-%Y") %> at <%= post.created_at.strftime("%I:%M:%S") %> by <%= reply.author.username %></small></div>
+        </div>
```

Comment:
This could be something you extract into a [View partial](https://api.rubyonrails.org/classes/ActionView/PartialRenderer.html), if you think it might be reused later on.